_posts/2020-12-01-infrastructure-migration-finished.md

```diff
@@ -0,0 +1,274 @@
+---
+title: Infrastructure migration finished
+author: TrueBrain
+---
+
+Today is a good day.
+Today is the day all services of openttd.org run on [AWS](https://aws.amazon.com/).
+Today is the day I can finally shut down our [OVH dedicated server](https://www.ovhcloud.com/en-gb/bare-metal/).
+Today is the day I can finally retire software like:
+- [Django](https://www.djangoproject.com/) 1.2 (released May 2010)
+- [Debian](https://www.debian.org/) Etch (release April 2007)
+- An IRC bot that was running since 2016.
+
+This was the longest migration I ever did: 11 months in the making.
+But we finally did it!
+
+This was mostly made possible to us receiving [AWS Open Source credits](https://aws.amazon.com/blogs/opensource/aws-promotional-credits-open-source-projects/).
```

Comment:
> This was mostly made possible to us receiving

This is a bit confusing, it should probably be "This was mostly made possible by us receiving"